lib/templates/courant/courantbeheer.tpl

```diff
@@ -62,6 +62,7 @@
 		<input type="button" value="Voorbeeld" onclick="CsrBBPreview('courantBericht', 'bewerkPreview');" />
 		{if $courant->magBeheren()}
 			<input type="button" value="Importeer agenda" onclick="importAgenda('courantBericht');" />
+			<input type="button" value="Importeer sponsor" onclick="document.getElementById('courantBericht').value += '[img]http://i.imgur.com/PYmPtiK.png[/img]'" />
```

Comment:
Misschien beter om hier ook een functie van te maken.

Reply:
http**s**

Reply:
Kunnen we niet beter https://csrdelft.nl/plaetjes/courant-sponsor.jpg doen?

Reply:
Nog beter: https://csrdelft.nl/plaetjes/sponsors/blah-blah-sponsor.jpg Het plaetje moet natuurlijk wel ergens blijven staan (voor een bepaalde tijd)